app/platform/templates/platform/developers/collection_details.html

```diff
@@ -17,6 +17,7 @@
 {% endblock beforeContent %}
 
 {% block content %}
+    <div class="fs-watermark">Developers</div>
```

Comment:
adding `aria-hidden="true"` would ask screen readers to ignore this, we'll be removing it before any users see it so I don't think this should be too big of an issue